controllers/nodemaintenance_controller.go

```diff
@@ -254,9 +254,8 @@ func createDrainer(ctx context.Context, mgrConfig *rest.Config) (*drain.Helper,
 	//Period of time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used.
 	drainer.GracePeriodSeconds = -1
 
-	// TODO - add logical value or attach from the maintenance CR
 	//The length of time to wait before giving up, zero means infinite
-	drainer.Timeout = DrainerTimeout
+	drainer.Timeout = evicitonTimeout
```

Comment:
what's going on with the old const now?

Reply:
Redundant and can be removed. The old value (30s) is the default value of the evicitonTimeout field.